pkg/adapter/adapter.go

```diff
@@ -119,6 +119,7 @@ func (l listener) handleEvent() http.HandlerFunc {
 			if err := json.Unmarshal(payload, &event); err != nil {
 				l.logger.Errorf("Invalid event body format format: %s", err)
 				response.WriteHeader(http.StatusBadRequest)
+				return
```

Comment:
is that fixing a bug?

Reply:
well the bug hasn't appeared yet ... it would happen if event payload is invalid json which is hard to happen
but need a return there.